src/rpc/transmission.ts

```diff
@@ -29,6 +29,8 @@ export const Status = {
     downloading: 4,
     queuedToSeed: 5,
     seeding: 6,
+    magnetizing: -1,
+    error: -2,
```

Comment:
This file should not be changed at all. It is meant to directly map to transmission API.

Once you move the status out of ProgressBar you will likely find that you don't need these fake statuses here.